content/en/docs/Reference/file-based-catalogs.md

```diff
@@ -539,6 +539,12 @@ The `olm.bundle.object` property [cue][cuelang-spec] schema is:
 [semver]: https://semver.org/spec/v2.0.0.html
 [semver-range]: https://github.com/blang/semver/blob/master/README.md#ranges
 
+#### `olm.deprecated`
+
+The `olm.deprecated` property is handled on the `olm.bundle` object such that if this property exists on the bundle object, the bundle is to be ignored and not installed. So, bundle deprecation was typically handled based on this property being present on the `olm.bundle` object. However, this was revelant for SQLite based catalogs which also required valid upgrade edges to be present to prevent foreign key violations.
+
+Similarly, the same goal is achieved for File-Based Catalogs simply by not including the bundle in the catalog; which effectively means that the `opm render` command will now exclude bundles that contain this property on the `olm.bundle` object. Although, there will be valid upgrade edges present even though the deprecated bundle and its metadata are excluded from the rendered File-Based Catalog.
```

Comment:
More simplification.  WDYT?

```suggestion
For file-based catalogs, we can simply omit the `olm.bundle` and maintain a valid upgrade graph.  Since this property no longer serves any function in file-based catalogs, `opm render` will exclude `olm.bundle`s that contain this property.  
```

Reply:
Looks good to me. Addressed in commit [5fe6f6c](https://github.com/operator-framework/olm-docs/pull/274/commits/5fe6f6c9b393b2c19fff0df77a4608c8cd376e68)